package.json

```diff
@@ -6,14 +6,18 @@
   "repository": "https://github.com/BrightspaceUI/testing.git",
   "scripts": {
     "lint": "eslint . --ext .js",
-    "test": "npm run lint && npm run test:headless",
-    "test:headless": "web-test-runner --files \"./test/**/*.test.js\" --node-resolve --playwright"
+    "test": "npm run lint && npm run test:server",
+    "test:headless": "web-test-runner --files \"./test/browser/**/*.test.js\" --node-resolve --playwright",
+    "test:server": "mocha ./test/server/**/*.test.js"
```

Comment:
Oh also, we still need to update `ci.yml` so these tests are actually running

Reply:
That's done [here](https://github.com/BrightspaceUI/core/pull/3711), if you want to review that one too.

Reply:
But don't we want to run the tests you've written here in this repo's CI?